app/src/main/res/values/strings.xml

```diff
@@ -137,6 +137,8 @@
     <string name="snackbar_viewAction">View</string>
     <string name="more_options_text">More Enhancement Options</string>
     <string name="delete">Delete</string>
+    <string name="snackbar_no_duplicate_pdf">Duplicate pages not found.No new PDF created.</string>
```

Comment:
add the translation of these strings to other locale files also. The error will be resolved then.
It is very clearly mentioned in the travis link also
https://travis-ci.org/Swati4star/Images-to-PDF/builds/470145810#L2057